Assets/Scripts/Minigames/CowardMiniGame/GearBehavior.cs

```diff
@@ -0,0 +1,126 @@
+/*****************************************************************************
+// File Name :         GearBehavior.cs
+// Author :            Mark Hanson
+// Contributors :      Marissa Moser
+// Creation Date :     5/24/2024
+//
+// Brief Description : Any function to do for the gears mini game will be found here. Includes swapping slots, Correct slot pattern with all bad ones, and selecting gears for each slot.
+*****************************************************************************/
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using PlaceboEntertainment.UI;
+
+public class GearBehavior : MonoBehaviour, IInteractable
+{
+    [Header("Individual Gear")]
+    [SerializeField] private GameObject[] _gearSize;
+    [SerializeField] private GameObject _gearIndi;
+    private int _gearSizeNum;
+    //private bool _scrollable;
+    private bool _doOnce;
+    //private PlayerController _pc;
+    private bool _interact;
+
+    [Header("Correct Gear")]
+    [SerializeField] private int _rightGearNum;
+    [SerializeField] private Color _matRed;
+    [SerializeField] private Color _matGreen;
+    private Renderer _rndr;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        _doOnce = true;
+        _gearSizeNum = 1;
+        //_scrollable = false;
+        //GameObject _pcObject = GameObject.FindWithTag("Player");
+        //_pc = _pcObject.GetComponent<PlayerController>();
+        _rndr = this.GetComponent<Renderer>();
+    }
+
+    // Update is called once per frame
+    void FixedUpdate()
```

Comment:
Consider adding more detailed comments to these update functions to describe what specifically they do

Reply:
added a bit more detail